clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractStickyAssignor.java

```diff
@@ -211,6 +211,10 @@ public boolean isSticky() {
         return partitionMovements.isSticky();
     }
 
+    public Map<TopicPartition, String> getPartitionsTransferringOwnership() {
```

Comment:
Forgot to mention, it's not a big deal but typically we don't include the "get" in getter names in the Kafka clients. Just a naming convention. ie this would be just `partitionsTransferringOwnership()`

Reply:
I know, plz check again